index.ts

```diff
@@ -127,6 +128,27 @@ if (!oauthEnabled) {
   });
 }
 
+app.get("/qr/:type/:room?", async (req: express.Request, res: express.Response) => {
+  const options: QRCode.QRCodeToBufferOptions = {
+    width: 300,
+    color: {
+      dark: "000000ff",
+      light: "ffffff00"
```

Comment:
The background color is white, but transparent. A solid white background better allows the image to be copied:
```suggestion
      light: "ffffff"
```

Reply:
According to the documentation it needs to be an rgba-value, but my initial reasoning for the became invalid by introducing the sharing modal.